src/app/snippet/components/snippet-card/snippet-card.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="section-snippet-card">
+
+  <h3>{{ snippet.title }}</h3>
+
+  <p class="comment">{{ snippet.comment }} </p>
+
+  <div class="labels" *ngIf="snippet.labels !== undefined">
```

Comment:
```suggestion
  <div class="labels" *ngIf="snippet.labels && snippet.labels.length">
```
tu vérifies en plus que la taille est supérieure à 0